scriptmodules/emulators/dosbox-sdl2.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="dosbox-sdl2"
+rp_module_desc="DOS emulator"
+rp_module_help="ROM Extensions: .bat .com .exe .sh .conf\n\nCopy your DOS games to $romdir/pc"
+rp_module_licence="GPL2 https://sourceforge.net/p/dosbox/code-0/HEAD/tree/dosbox/trunk/COPYING"
+rp_module_section="exp"
+rp_module_flags="dispmanx !mali !kms"
+
+function depends_dosbox-sdl2() {
+    local depends=(libsdl2-dev libsdl2-net-dev libasound2-dev libpng12-dev automake autoconf zlib1g-dev subversion)
+    isPlatform "rpi" && depends+=(timidity freepats)
+    getDepends "${depends[@]}"
+}
+
+function sources_dosbox-sdl2() {
+    svn checkout https://svn.code.sf.net/p/dosbox/code-0/dosbox/trunk "$md_build" -r 4006
```

Comment:
Also can use the sources_dosbox ?

Reply:
LOL, didn't think of that for some reason. Will change it.

Reply:
Actually, I remember why I didn't do it. It might be a bad idea to share the function as it will lock us into r4006 for the vanilla port. We need to stick to that revision for the SDL2 patch to apply cleanly, but newer SVN builds actually works fine on Pi, and we might want to update the vanilla port later.